text.txt

```diff
@@ -1,8 +1,14 @@
 Vitajte v Progressbare!
 
-Progressbar je občianske združenie a priestor, v ktorom sa stretávajú hackeri, geekovia, umelci, nadšenci do elektroniky a dokonca aj psychológovia.
-Máme tu labák, prednáškovú miestnosť, coworking space a kuchynku. Ak by ste potrebovali ísť na WC, nájdete ho vzadu za kuchynkou.
+Progressbar je priestor, v ktorom sa stretávajú ľudia rôznych povolaní, záujmov a kultúr. 
+Hackeri, geekovia, umelci, nadšenci do elektroniky, psychológovia a mnoho iných.
+Máme tu laboratórium, prednáškovú miestnosť, coworking space a kuchynku. 
+Toaleta sa nachádza vzadu za kuchynkou.
 
-V kuchynke funguje samoobslužný systém. Môžete si zobrať na čo máte chuť a prispieť poslaním Bitcoinov alebo vhodením Eur do pokladničky.
+V kuchynke funguje samoobslužný systém. 
+Môžete si zobrať na čo máte chuť a prispieť poslaním Bitcoinov alebo vhodením Eur do pokladničky.
 
-Ako občianske združenie fungujeme z členského a dobrovoľných príspevkov návštevníkov. Poteší nás každý príspevok a ak sa budete chcieť stač členom, môžete sa zaregistrovať na našej stránke. Všetci sú u nás vítaní!
+Ako občianske združenie fungujeme z členského a dobrovoľných príspevkov návštevníkov. 
+Poteší nás každý príspevok a nový členovia sú vždy vítaní!
+
+Viac informácií nájdete na https://www.progressbar.sk/pridaj-sa
```

Comment:
Zda sa mi, ze ten clanok rozobera skor to, ako si skonfigurovat server a nie ako komunikovat adresu ludom. Web Progressbaru dokaze obidve verzie.

Z jeho dovodov jediny teoreticky relevantny je asi tento (zvyraznil som dolezitu cast):

> The www makes a hostname visually self-identify as a web address, without needing an ugly "http://". This is especially appealing in printed marketing materials such as business cards, TV ads etc. **This may be even more important for uncommon top-level domain names (those other than .com/.org/.net), especially newly created ones and country TLDS that are used for vanity domains, which non-expert users may not recognise.** Note also that some browser (e.g. Google Chrome) hide the "http://", so it may fall further out of use.

Osobne si myslim, ze kedze video je v Slovencine, tak je targetovane na slovenske publikum a teda mozno predpokladat, ze dokaze pochopit, ze ide o URL. Toto mozno doplnit whodnym obrazkom (kurzor mysi a pod).

Osobne oblubujem jednoduchost, preto by som preferoval `progressbar.sk`.

V pripade, ze by sme chceli aj EN verziu, asi by som si vedel predstavit aj www.